src/plugins/intellisense/index.tsx

```diff
@@ -60,12 +61,22 @@ export function getExpressionLatex(id: string): string | undefined {
   ).latex;
 }
 
-export default class Intellisense extends PluginController {
+export default class Intellisense extends PluginController<{
+  subscriptify: boolean;
+}> {
   static id = "intellisense" as const;
   static enabledByDefault = false;
   static descriptionLearnMore =
     "https://github.com/DesModder/DesModder/tree/main/src/plugins/intellisense/docs/README.md";
 
+  static config = [
+    {
+      type: "boolean",
+      key: "subscriptify",
+      default: false,
```

Comment:
It would be cool if we could somehow mark that this option pairs well with custom-mathquill-config's option to not auto-subscript digits. Separate PR though.